activities/SharedNotes.activity/locale.ini

```diff
@@ -46,7 +46,7 @@ PlayerJoin={{user}} joined
 PlayerLeave={{user}} left
 
 [en]
-NotesSaved=Image was saved to journal!
+NotesSaved = Your Notes image was exported
```

Comment:
This update is not related to this fix.

Reply:
I know its a conflict I realised later after the push sorry